# params/zed2_params.yaml
# Common ROS parameters to the object detection code
---

zed_node: {
    # loop rate
    loop_rate:          5,
    
    # the location of the network weights file
    net_file:           "/home/jax/catkin_ws/src/robot/common/nets/dc3_rgb_v10e_035_035_100_90_HPC_final_net.dat",
    
    # image cropping parameters
    crop_x:             128,
    crop_y:             196,
    crop_w:             1024,
    crop_h:             512, 
    
}

